.github/workflows/cmake.yml

```diff
@@ -90,9 +90,9 @@ jobs:
            ${{ matrix.cmake_options }}
            -DCMAKE_INSTALL_PREFIX=${{ matrix.install_dir }}
            -DCMAKE_BUILD_TYPE=${{ env.cmake_build_type }}
-           -DSKELETON=ON
+           -DPA_USE_SKELETON=ON
            -DBUILD_TESTING=ON
```

Comment:
I'm curious about why PA_BUILD_EXAMPLES is prefixed and BUILD_TESTING is not. Is BUILT_TESTING a CMake convention?

Reply:
Yes, BUILD_TESTING is a CMake convention.

Reply:
Isn't `BUILD_TESTING` variable/option set by CMake when only CTest is used?
https://cmake.org/cmake/help/v3.1/module/CTest.html

Currently according to CMakeLists.txt `BUILD_TESTING` is used as local option:

**set here**
https://github.com/PortAudio/portaudio/blob/06eab6074b74a3d8bd155bed714105707337c1e8/CMakeLists.txt#L9

**applied here**
https://github.com/PortAudio/portaudio/blob/06eab6074b74a3d8bd155bed714105707337c1e8/CMakeLists.txt#L364-L379

thus to my view here we have ambiguous symbol because CTest is not used by the script and it needs to be fixed by prefixing with PA_ to become `PA_BUILD_TESTING` or better `PA_BUILD_TESTS`.